tests/lib/send_results.sh

```diff
@@ -0,0 +1,202 @@
+#!/bin/bash
+
+# This script will store the results into csv files in the
+# metrics 'results' directory.
+# The csv file names are derived from the reported 'test name', and all
+# results for a single 'test name' are collated (appended) into a single csv.
+
+# General env
+MYSELF="${0##*/}"
+SCRIPT_PATH=$(dirname $(readlink -f $0))
+RESULT_DIR="${SCRIPT_PATH}/../metrics/results"
+GROUP="PNP"
+
+declare TEST_NAME
+declare RESULT
+declare ARGS
+declare UNITS
+declare SYS_VERSION
+declare SYSTEM
+declare TAG
+declare HW
+declare SEND
+declare KERNEL
+declare IMG
+
+# Parse the platform name
+function get_platform_name()
+{
+	com="core"
+
+	model="$(cat /proc/cpuinfo | grep "model name" | uniq | \
+		awk -F ": " '{print $2}' | sed s/\(R\)//g | \
+		sed s/\(TM\)//g | sed s/[[:space:]]*CPU[[:space:]]*/" "/g | \
+		cut -d"@" -f1 | sed s/[[:space:]]*$//g)"
+
+	num_cores=$(nproc)
+
+	if (( $num_cores > 1 ));then
+		com="cores"
+	fi
+
+	echo "$model ($num_cores $com)"
+}
+
+function save_to_csv()
+{
+	runtime="/usr/bin/cc-oci-runtime"
+	release_file="/usr/lib/os-release"
+	timestamp="$(date +%s)"
+	cc_img_path="/usr/share/clear-containers"
+
+	if [ -z "$TEST_NAME" ];then
+		echo "error: test name no found"
+		exit 1;
+	fi
+
+	if [ -z "$RESULT" ];then
+		echo "error: result no found"
+		exit 1;
+	fi
+
+	if [ -z "$UNITS" ];then
+		echo "error: units no found"
+		exit 1;
+	fi
+
+	if [ -z "$HW" ];then
+		platform="$(get_platform_name)"
+		HW="$platform"
+	fi
+
+	if [ -z "$IMG" ];then
+		IMG="$(readlink "$cc_img_path/clear-containers.img")"
+	fi
+
+	if [ -z "$KERNEL" ];then
+		KERNEL="$(readlink "$cc_img_path/vmlinux.container")"
```

Comment:
I think that is true @MarioCarrilloA , and I am coding a bit of 'backup' into the script right now - but, the ultimate code would locate exactly what runtime the test ran with, and then extract the info about the img file and kernel from that runtime setup. Sadly, we are some way from coding that up today - so, today we will go with @xxx@, and 'adapt'. It may be for now that the person or system who runs the tests may have to set some ENV or command line vars to point to the correct img and kernel files.